layers/land.yaml

```diff
@@ -80,7 +64,8 @@ layers:
                 extrude: true

```

Comment:
You can filter extrude = true this will remove some edge cases where outer building will hide building parts

Reply:
Wouldn't this hide all buildings without height information?

Reply:
Hum.... Good point... In my current version when `extrude=false`, height is not null... I will remove height when extrude is false, this is more.... logic :sweat_smile: